hdtv/plugins/dblookup.py

```diff
@@ -46,80 +45,85 @@ def __init__(self):
         # Register configuration variables for fit peakfind
         self.opt["db"] = hdtv.options.Option(
             default="PGAAlib_IKI2000",
-            parse=hdtv.options.parse_choices(
-                list(hdtv.database.databases.keys())),
-            changeCallback=lambda x: self.Set(x))  # default database
+            parse=hdtv.options.parse_choices(list(hdtv.database.databases.keys())),
+            changeCallback=lambda x: self.Set(x),
+        )  # default database
         hdtv.options.RegisterOption("database.db", self.opt["db"])
 
         # Automatically lookup fitted peaks in database
         self.opt["auto_lookup"] = hdtv.options.Option(
             default=False,
             parse=hdtv.options.parse_bool,
-            changeCallback=lambda x: self.SetAutoLookup(x))
-        hdtv.options.RegisterOption(
-            "database.auto_lookup", self.opt["auto_lookup"])
+            changeCallback=lambda x: self.SetAutoLookup(x),
```

Comment:
Codacy found an issue: [Lambda may not be necessary](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)